linux_os/guide/services/ssh/ssh_server/sshd_set_maxstartups/rule.yml

```diff
@@ -0,0 +1,34 @@
+documentation_complete: true
+
+title: 'Ensure SSH MaxStartups is configured'
+
+description: |-
+    The MaxStartups parameter specifies the maximum number of concurrent
+    unauthenticated connections to the SSH daemon. Additional connections will be
+    dropped until authentication succeeds or the LoginGraceTime expires for a
+    connection. To confgure MaxStartups, you should add or correct the following
+    line in the
+    <tt>/etc/ssh/sshd_config</tt> file:
+    <pre>MaxStartups {{{ xccdf_value("var_sshd_set_maxstartups") }}}</pre>
```

Comment:
Technically, for CIS, it also includes this note:

```
> Note: Local site policy may be more restrictive.
```

But it seems like you're making this "equals" and not allowing "more restrictive" in the OVAL -- just something to discuss. What are your thoughts on the "or more restrictive" clause? Even if the OVAL doesn't, if someone was just reading the XCCDF it might be nice to mention it.

Reply:
This is a good point. I've added wording to this effect into the rule and will investigate strengthening the OVAL check also.